zeeguu/api/api/nlp.py

```diff
@@ -58,3 +59,28 @@ def annotate_clues():
 
     return json_result(updated_words)
 
+# ---------------------------------------------------------------------------
+@api.route("/get_sentences_for_wo", methods=("POST",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def get_sentences_for_wo():
+    article_text = request.form.get("article_text", "")
+    bookmark_context = request.form.get("bookmark_context", "")
+    language = request.form.get("language")
+
+    if language not in SpacyWrappers.keys():
+        return "Language not supported"
+    
+    nlp_pipe = SpacyWrappers[language]
+    sentences = nlp_pipe.get_sent_list(article_text)
+    filtered_sentences = [sent for sent in sentences if len(nlp_pipe.tokenize_sentence(sent)) <= 15]
+    
+    context_doc = nlp_pipe.spacy_pipe(bookmark_context)
+    heap = []
+    for f_sent in filtered_sentences:
```

Comment:
I see what's confusing to me now.

`nlp_pipe.spacy_pipe` sounds confusing. Could we maybe add a `process()` function in the SpacyWrapper, and optionally name the var above `spacy_pipe` instead of `nlp_pipe` such that the code will look like: `context_doc = spacy_pipe.process(bookmark_context)`? 

That way this code will read easier I think.